hack/build/print-workspace-status.sh

```diff
@@ -47,6 +47,8 @@ gitTreeState ${KUBE_GIT_TREE_STATE-}
 gitVersion ${KUBE_GIT_VERSION-}
 gitMajor ${KUBE_GIT_MAJOR-}
 gitMinor ${KUBE_GIT_MINOR-}
+gitPatch ${KUBE_GIT_PATCH-}
+lastRelease ${KUBE_LAST_RELEASE}
```

Comment:
I don't think these new bazel variables are used anywhere. 
Can they be removed?